test/scheduler.test.ts

```diff
@@ -130,4 +130,237 @@ describe("Schedule Store", () => {
   test("delete non-existing schedule", async () => {
     await expect(store.schedules.delete("non-existing-id")).rejects.toThrow();
   });
+
```

Comment:
Do you think we still need the tests above? If they are redundant, let's git rid of them.

Also, can we delete the schedule(s) at the end of each tests? This will come in handy for the remote store.